unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/Sections/PlacesAndEventsSection/SubSections/HighlightsSubSection/HighlightsSubSectionMenu/HighlightsSubSectionComponentView.cs

```diff
@@ -239,17 +260,16 @@ public void SetLiveEvents(List<EventCardComponentModel> events)
         SetLiveAsLoading(false);
         liveEventsNoDataText.gameObject.SetActive(events.Count == 0);
 
+        setLiveEventsTask = (ct) => SetLiveEventsAsync(events, liveEvents, liveEventCardsPool, disposeCts.Token);
```

Comment:
Not very common but interesting: instead of calling a method directly with a local field, call a delegate with capture. Do you think it is a good approach?

Reply:
Nope, I hate it. It's some sort of "time slicing" approach (you can take a look at `UpdateCardsVisualProcess` at the bottom for more context).

I'd change it but the refactor would be huge and Im afraid of breaking a million things in the process.